Assets/HoloToolkit/Input/Scripts/Interactable.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Base class Interactable GameObjects can inherit from.
+    /// </summary>
+    public class Interactable : MonoBehaviour
+    {
+        protected bool TriggerKeywordOnGazeOnly;
+
+        protected bool IsFocused;
+
+        protected virtual void OnEnable()
+        {
+            GestureManager.Instance.OnTap += OnTap;
+            GazeManager.Instance.OnGazeExit += OnGazeExit;
+            GazeManager.Instance.OnGazeEnter += OnGazeEnter;
+            KeywordManager.Instance.OnKeywordRecognized += KeywordRecognized;
+        }
+
+        protected virtual void OnDisable()
+        {
+            GestureManager.Instance.OnTap -= OnTap;
+            GazeManager.Instance.OnGazeExit -= OnGazeExit;
+            GazeManager.Instance.OnGazeEnter -= OnGazeEnter;
+            KeywordManager.Instance.OnKeywordRecognized -= KeywordRecognized;
+        }
+
+        /// <summary>
+        /// Called when a user has tapped any gameObject.
+        /// </summary>
+        /// <param name="tappedGameObject">GameObject user has tapped.</param>
+        protected virtual void OnTap(GameObject tappedGameObject)
+        {
+            if (tappedGameObject == gameObject)
```

Comment:
The script you have, all you need to do is update the 'MaxListeners' to set the number of callers who register.  FWIW, TimeSpan (the result of DateTime-DateTime) has a ticks property as well.  Using the stop watch is fine of course. :)

I looked at the changes you made to my scripts.  The signal to receive time is interesting, but for the event test the results are going to vary quite a bit, and the debug.log in the callback should make it so that by the time the last object is signaled the time is really large.  With 100 subscribers, I would expect that you get 100 lines of 'SendEvent current total ...' that get progressively longer.